src/bikeracks/templates/scripts.js

```diff
@@ -48,6 +49,13 @@ class BikeMap {
             this.tempMarker.remove();
         }.bind(this));
         
+        // suggest removal click handler
+        this.$suggestRemoval.on('click', function(e) {
+            const button = $(event.target),
+                  rack_id = button.data('rack_id');
+            this.suggestRemoval(event, rack_id);
```

Comment:
Same question here. `e` instead of `event` ?

Reply:
Yes, that is what I meant.